flow/data_pipeline/datapipeline_test.py

```diff
@@ -0,0 +1,37 @@
+"""functions that calculates the expected result for testing."""
+import math
+
+# Vehicle Mass
+M = 1200
+# Gravity
+g = 9.81
+# Density of Air
+ro_air = 1.225
+# Rolling resistance coefficient
+C_r = .005
+# Aerodynamic drag coefficient
+C_a = 0.3
+# Vehicle Cross sectional Area
+A = 2.6
+# Road grade
+theta = 0
+
+
+def heavyside(inp):
+    """Return 1 if input is positive."""
+    return 0 if inp <= 0 else 1
+
+
+def calculate_power(mu, acceleration, M=M, g=g, theta=theta, C_r=C_r, ro_air=ro_air, A=A, C_a=C_a):
```

Comment:
See below comment about reimplementing the power demand function here

Reply:
replied below, but not sure what the alternative is. i think this function just exists here for the sake of producing a test output (so do we remove it?).

it will however be re-implemented in another PR by the energy team, so that the RL team can compute energy rewards on-the-fly. are we ok with that duplication?

Reply:
If it's for unit tests move it to the unit test folder?